src/components/AvatarGroup/AvatarGroup.jsx

```diff
@@ -0,0 +1,144 @@
+import React, { useCallback } from "react";
+import PropTypes from "prop-types";
+import cx from "classnames";
+import Avatar from "../Avatar/Avatar";
+import Counter from "../Counter/Counter";
+import Tooltip from "../Tooltip/Tooltip";
+import AvatarGroupCounterTooltipContainer from "./AvatarGroupCounterTooltipContainer";
+import styles from "./AvatarGroup.module.scss";
+
+const AvatarGroup = ({
+  className,
+  id,
+  children,
+  size,
+  type,
+  max,
+  counterProps,
```

Comment:
I don't think we want to add support on this customization - its different from tooltip because we do want all counter will look the same (in tooltip we maybe want to allow to open it from different places or allow different triggers for close it).
it also can allow people to create weird appearance for example change the color of the counter without change the wrapper.

Reply:
With `counterProps` I've replaced 4 props: `counterValue`, `counterColor`, `counterPrefix`, `counterMaxDigits`. I thought it would be better this way due to similarity with `counterTooltipCustomProps` and fewer amount of props
But for now only these 4 props are actually passing into Counter, the rest is not used (though it might be a bit misguiding)

Didn't get the example with counter color change without wrapper change, let's discuss